### PR TITLE
vtx_osd_variant_fix

### PR DESCRIPTION
**Pull request by P-I-Engineer**
this is a PR to replace https://github.com/betaflight/betaflight-configurator/pull/2755 i had too many github issues to salvage

I've created an osd variant for VTX channel that displays only the VTXpower.

pull request is partial approved in betaflight here
https://github.com/betaflight/betaflight/pull/11275

image
image
image

### CODE REVIEW

File: src/js/tabs/osd.js
Comment (haslinghuis) [inline src/js/tabs/osd.js:586]: Please fix indentation.
```suggestion
           ],
```

File: src/js/tabs/osd.js
Comment (haslinghuis) [inline src/js/tabs/osd.js:589]: Missing comma:
```suggestion
            },
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

**Comment by P-I-Engineer** [comment]
i know this is messed up again, i'm having a heck of a time with squashing commits

**Review by haslinghuis — APPROVED** [review]

**Comment by haslinghuis** [comment]
@P-I-Engineer please check https://github.com/betaflight/betaflight/wiki/Betaflight-Contribution. You need to master this.

**Comment by P-I-Engineer** [comment]
i keep getting merge commits and not sure where they're coming from

**Comment by P-I-Engineer** [comment]
got it

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Pull request merged**
